src/screens/Settings/SelectFiatUnit.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+import React, { Component } from "react";
+import { View, FlatList } from "react-native";
+import { connect } from "react-redux";
+import { listFiats } from "@ledgerhq/currencies";
+import UnitRow from "../../components/UnitRow";
+import { saveSettings } from "../../actions/settings";
+import type { SaveSettings } from "../../actions/settings";
+
+const mapDispatchToProps = {
+  saveSettings
+};
+
+class SelectFiatUnit extends Component<{
+  navigation: *,
+  saveSettings: SaveSettings
+}> {
+  static navigationOptions = {
+    title: "Countervalue currency"
+  };
+
+  onPress = (item: *) => {
+    const { navigation, saveSettings } = this.props;
+
+    saveSettings({ counterValue: item.code });
+    navigation.goBack();
+  };
+
+  renderItem = ({ item }) => (
+    <UnitRow unit={item} onPress={() => this.onPress(item)} />
+  );
+
+  keyExtractor = item => item.code;
+
+  render() {
+    const fiatList = listFiats();
+
+    fiatList.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
two problem:

- sort() is mutable. so it will mutate the list returned by listFiats(). if another code somewhere where using that fiatList, it would affect it because I think i don't reclone a new array in its implementation.
- it is done at each time of a render()

so probably you can do `const fiatList = listFiats().slice(0)..sort((a, b) => a.name.localeCompare(b.name))` as a defaultProps , or in a global variable. but not inside render()